proposals/dao/fip_15.js

```diff
@@ -0,0 +1,119 @@
+/* eslint-disable max-len */
+/* eslint-disable no-empty-function */
+/* eslint-disable no-unused-vars */
+const { ethers } = require('hardhat');
+const hre = require('hardhat');
+const { BN } = require('../../test/helpers');
+
+const {
+  coreAddress,
+  tribeAddress,
+  feiRewardsDistributorAddress,
+  curve3Metapool,
+  feiTribePairAddress
+} = require('../../contract-addresses/mainnetAddresses.json');
+
+const Tribe = artifacts.require('Tribe');
+const Core = artifacts.require('Core');
+
+const e18 = '000000000000000000';
+const seventyFiveTribe = `75${e18}`;
+const twoMillionTribe = `2000000${e18}`;
+const allocPoints = 1000;
+const zeroMultiplier = '10000';
+const ZERO_ADDRESS = '0x0000000000000000000000000000000000000000';
+
+const defaultPoolRewardObject = [
+  {
+    lockLength: 0,
+    rewardMultiplier: zeroMultiplier,
+  },
+];
+
+async function setup(addresses, oldContracts, contracts, logging) {
+  const { timelockAddress } = addresses;
+
+  await hre.network.provider.request({
+    method: 'hardhat_impersonateAccount',
+    params: [timelockAddress],
+  });
+}
+
+// assert that state changes correctly
+// add the first couple of pools to the tribal chief
+// add pool for curve staking, fei/tribe on univ2, rari tribe pool
+// add tests to the validate function. Just test things like balances, allocation points, etc
+async function run(addresses, oldContracts, contracts, logging = false) {
+  const { stakingTokenWrapper, tribalChief } = contracts;
+  const { timelockAddress } = addresses;
+
+  const tribe = await Tribe.at(tribeAddress.address);
+  const core = await Core.at(coreAddress.address);
+
+  // we should subtract 2 million off this number to leave 2 million tribe in the DAO
+  const tribeBalanceToMigrate = await tribe.balanceOf(feiRewardsDistributorAddress.address);
+
+  const governorWithdrawTribeAbi = ['function governorWithdrawTribe(uint256 amount)'];
+  const governorWithdrawTribeInterface = new ethers.utils.Interface(governorWithdrawTribeAbi);
+  const encodedGovernorWithdrawTribe = governorWithdrawTribeInterface.encodeFunctionData('governorWithdrawTribe', [tribeBalanceToMigrate.toString()]);
+  const adminSigner = ethers.provider.getSigner(timelockAddress);
+  await (
+    await adminSigner.sendTransaction({ data: encodedGovernorWithdrawTribe, to: feiRewardsDistributorAddress.address })
+  ).wait();
+    
+  const tribeBalanceToAllocate = tribeBalanceToMigrate.sub(new BN(twoMillionTribe));
+  // distribute 75 tribe per block instead of 100
+  await tribalChief.updateBlockReward(seventyFiveTribe);
+  // then migrate the balance minus 2 million to the tribalchief
+  await core.allocateTribe(tribalChief.address, tribeBalanceToAllocate);
```

Comment:
ok this should be the last scope increase but we don't want to send all the TRIBE at once. There should be some kind of dripper between the treasury and the tribalChief to minimize the attack surface. I think we can use a basic PCV deposit with a PCVDripController to do this

Reply:
I'll have to go read into and understand how to wire those contracts up. What are you thinking for drip frequency and amounts? Will it be a call that is incentivized, or not?

Reply:
Notes:
Not incentivized, weekly frequency with tribe send big enough to cover at least 1.5 weeks.

Reply:
This will be a separate PR.